cmd/tikv-ctl/src/main.rs

```diff
@@ -235,6 +241,75 @@ fn main() {
             let key = unescape(&key);
             split_region(&pd_client, mgr, region_id, key);
         }
+        Cmd::ShowClusterId { data_dir } => {
+            if opt.config.is_none() {
+                eprintln!("tikv-ctl parameter config must be specified");
+                process::exit(-1);
+            }
+            if data_dir.is_empty() {
+                eprintln!("reuse-readonly-remains parameter data-dir must be specified");
+                process::exit(-1);
+            }
+            cfg.storage.data_dir = data_dir;
+            // Disable auto compactions and GCs to avoid modifications.
+            cfg.rocksdb.defaultcf.disable_auto_compactions = true;
+            cfg.rocksdb.writecf.disable_auto_compactions = true;
+            cfg.rocksdb.lockcf.disable_auto_compactions = true;
+            cfg.rocksdb.raftcf.disable_auto_compactions = true;
+            cfg.raftdb.defaultcf.disable_auto_compactions = true;
+            cfg.rocksdb.titan.disable_gc = true;
+            match read_cluster_id(&cfg) {
+                Ok(id) => {
+                    println!("cluster-id: {}", id);
+                    process::exit(0);
+                }
+                Err(e) => {
+                    eprintln!("read cluster ID fail: {}", e);
+                    process::exit(-1);
+                }
+            }
+        }
+        Cmd::ReuseReadonlyRemains {
```

Comment:
shall we set `raftstore::store::config::snap_apply_copy_symlink` to true by default?

Reply:
Generally snapshot files are generated by TiKV, so they should be regular files instead of symlinks. So it's ok to set `snap_apply_copy_symlink` to `false` in general cases.
By design, only TiKV instances created by `reuse-readonly-remains` need to set the flag.